scripts/ci/install

```diff
@@ -14,10 +14,14 @@ def run(command):
     return check_call(command, shell=True)
 
 
-run('pip install --no-build-isolation -r requirements-base.txt')
-run('pip install --no-build-isolation -r requirements.txt')
-if os.path.isdir('dist') and os.listdir('dist'):
-    shutil.rmtree('dist')
-run('python -m build')
-wheel_dist = glob.glob(os.path.join('dist', '*.whl'))[0]
-run('pip install %s' % wheel_dist)
+run("pip install setuptools==57.5.0")
```

Comment:
Let's make sure to remove this direct install and defer to our requirements files for dependencies.